tox.ini

```diff
@@ -9,8 +9,8 @@ deps=
 commands=
   pytest {posargs}
 setenv =
-  ORIGO_API_CLIENT = my-origio-user
-  ORIGO_API_PASSWORD = my-origo-password
+  ORIGO_API_aCLIENTz = my-origio-user
```

Comment:
Forgot to set them back to their original value?

Reply:
Yes, was testing if these where actually needed